ct.yaml

```diff
@@ -4,12 +4,10 @@
 remote: origin
 target-branch: main
 chart-dirs:
-  - chart
```

Comment:
`ct` expects a structure similar to `charts/<CHART>/Chart.yaml` and since we are not following this and using a structure of `chart/Chart.yaml` then setting `chart-dirs` to top level directory (`.`) seems to work correctly and `ct` is able to detect chart changes.